include/mapbox/geojsonvt/clip.hpp

```diff
@@ -186,7 +189,7 @@ class clipper {
         }
 
         if (!slice.empty()) { // add the final slice
-            slice.segEnd = lineLen;
+            if (lineMetrics) slice.segEnd = lineLen;
```

Comment:
🤔 that was a bug, right? We were not supposed to update `slice.segEnd` if line metrics is disabled.

Reply:
yes